tests/e2e-leg-6/verify-server-logrotate/sql-input/logrotate.sql

```diff
@@ -0,0 +1,35 @@
+-- Cleanup the DC table
```

Comment:
Can you first check the config param `EnableLogrotate`? So that if it is not 1 it fails here. 
If one query  returns an error are we still going to run the rest of them? If yes can we exit once one query fails?

Reply:
1. We don't need to do it; select hurry_service('SYSTEM', 'LogRotate', 60) will handle it. If we don't set the config parameter EnableLogrotate to 1, it will fail immediately at hurry_service.
2. Yes, I can use one more if-else condition to run queries. If any queries fail, we can return an error.